ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml

```diff
@@ -25,9 +21,6 @@ images:
     ocp_4.15_base-rhel9:
       as:
       - registry.ci.openshift.org/ocp/4.15:base-rhel9
-    ocp_builder_rhel-9-golang-1.20-openshift-4.15:
-      as:
-      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15
```

Comment:
Can you remove this whole `inputs` section?
Basically revert the changes the bot made for these files in https://github.com/openshift/release/commit/fa388b204f7d2282d36805de37058216f0336621
```
--- a/ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml
+++ b/ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml
@@ -3,6 +3,14 @@ base_images:
     name: commitchecker
     namespace: ci
     tag: "4.15"
+  ocp_4.15_base-rhel9:
+    name: "4.15"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.20-openshift-4.15:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.15
   tests-private:
     name: tests-private
     namespace: ci
@@ -13,6 +21,13 @@ build_root:
 canonical_go_repository: sigs.k8s.io/ibm-vpc-block-csi-driver
 images:
 - dockerfile_path: Dockerfile.openshift
+  inputs:
+    ocp_4.15_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.15:base-rhel9
+    ocp_builder_rhel-9-golang-1.20-openshift-4.15:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15
   to: ibm-vpc-block-csi-driver
 promotion:
   name: "4.15"
```

Reply:
Same for the operator as well

Reply:
Sure, I can do that.

Reply:
I think I got them updated that way.